src/Provider/Provider.Web/Orchestrators/DashboardOrchestrator.cs

```diff
@@ -99,43 +99,40 @@ private int SanitizePage(int page, int totalVacancies)
             return null;
         }
 
-        private List<SelectListItem> GetFilterSelectOptions(List<VacancySummary> vacancies, VacancyStatus? filterStatus)
+        private List<SelectListItem> GetFilterSelectOptions(VacancyStatus? filterStatus)
         {
             return new List<SelectListItem>
             {
-                GetFilterSelectListItem(vacancies, null, filterStatus),
-                GetFilterSelectListItem(vacancies, VacancyStatus.Draft, filterStatus),
-                GetFilterSelectListItem(vacancies, VacancyStatus.Submitted, filterStatus),
-                GetFilterSelectListItem(vacancies, VacancyStatus.Live, filterStatus),
-                GetFilterSelectListItem(vacancies, VacancyStatus.Closed, filterStatus),
-                GetFilterSelectListItem(vacancies, VacancyStatus.Referred, filterStatus),
+                GetFilterSelectListItem(null, filterStatus),
+                GetFilterSelectListItem(VacancyStatus.Live, filterStatus),
+                GetFilterSelectListItem(VacancyStatus.Referred, filterStatus),
+                GetFilterSelectListItem(VacancyStatus.Submitted, filterStatus),
+                GetFilterSelectListItem(VacancyStatus.Draft, filterStatus),
+                GetFilterSelectListItem(VacancyStatus.Closed, filterStatus),
+                
             };
         }
 
-        private SelectListItem GetFilterSelectListItem(IEnumerable<VacancySummary> vacancies, VacancyStatus? optionStatus, VacancyStatus? filterStatus)
+        private SelectListItem GetFilterSelectListItem(VacancyStatus? optionStatus, VacancyStatus? filterStatus)
         {
-            var count = vacancies.Count(v => 
-                optionStatus.HasValue == false || 
-                v.Status == optionStatus.Value);
-
             var value = optionStatus.HasValue ? optionStatus.Value.ToString() : "All";
 
-            var text = optionStatus.HasValue ? optionStatus.Value.GetDisplayName() : "All Vacancies";
+            var text = optionStatus.HasValue ? optionStatus.Value.GetDisplayName().ToLowerInvariant() + " vacancies" : "all vacancies";
 
-            return new SelectListItem($"{text} ({count})", value, optionStatus == filterStatus );
+            return new SelectListItem($"{text}", value, optionStatus == filterStatus );
         }
 
         private string GetFilterHeading(int totalVacancies, VacancyStatus? filterStatus)
         {
             if (totalVacancies == 1 && filterStatus.HasValue == false)
-                return "Showing 1 vacancy";
+                return "Show 1 vacancy";
```

Comment:
if it is just 1 then should it not be `Show vacancy` only

Reply:
Does this text still need changing as per Sachin's comment?